FinSearchUnified/Helper/StaticHelper.php

```diff
@@ -533,8 +533,19 @@ private static function createRangeSlideFacet(array $facetItem, $min, $max, $act
     {
         $request = Shopware()->Front()->Request();
 
-        // Perform a loose comparison since the floating numbers could actually be integers.
-        if ($request->getParam('min') == $activeMin || $request->getParam('max') == $activeMax) {
+        $minFieldName = 'min' . $facetItem['name'];
+        $maxFieldName = 'max' . $facetItem['name'];
+
+        if ($facetItem['name'] === 'price') {
+            $minFieldName = 'min';
+            $maxFieldName = 'max';
+        }
+
+        if ($request->getParam($minFieldName) === null && $request->getParam($maxFieldName) === null) {
+            $active = false;
+            // Perform a loose comparison here since the floating numbers could actually be integers.
```

Comment:
The idea was that this `if` block is exactly structured like this:
```
if (is null) {
    $active = false;
}
// Perform a loose comparison since the floating numbers could actually be integers.
elseif (values match) {
    $active = true;
} else {
    $active = false
}
```
This way it will be clear where the comment belongs to. If this raises a code style error, I would propose to move the comment inside the `elseif`.